adops/gam-native.md

```diff
@@ -0,0 +1,221 @@
+
+![native ad styling](/assets/images/ad-ops/dfp-native/gam-native-template.png){: .pb-xlg-img :}
+
+{: .alert.alert-warning :}
+Any link that needs to fire a click tracker must include `class='pb-click'`.
+
+If this creative is served, it will fire impression trackers on load. Clicking the link will fire the click tracker and the link will work as normal, in this case going to the `hb_native_linkurl` destination.
+
+The creative template HTML will depend on which of the three scenarios you're implementing, as described in the [Native Implementation Guide](/prebid/native-implementation.html). You can choose to manage the native template:
+
+- in GAM ([section 3.1](/adops/gam-native.html#31-managing-the-native-template-in-gam) below)
+- in the Prebid.js AdUnit ([section 3.2](/adops/gam-native.html#32-managing-the-native-template-outside-of-gam) below)
+- in a separate JavaScript file ([section 3.2](/adops/gam-native.html#32-managing-the-native-template-outside-of-gam) below)
+
+### 3.1. Managing the native template in GAM
+
+There are three key aspects of the native template:
+
+1. Build the creative with special Prebid.js macros, e.g. `##hb_native_assetname##`. See the appendix for an exhaustive list of assets and macros. Note that macros can be placed in the body (HTML) and/or head (CSS) of the native creative.
+2. Load the Prebid.js native rendering code. You may utilize the jsdelivr version of native-render.js or host your own copy. If you use the version hosted on jsdelivr, make sure to declare jsdelivr as an ad technology provider in GAM. Admin → EU User Consent → Declare ad technology providers
+3. Invoke the Prebid.js native rendering function with an object containing the following attributes:
+    - adid - used to identify which Prebid.js creative holds the appropriate native assets
+    - pubUrl - the URL of the page, which is needed for the HTML postmessage call
+    - requestAllAssets - tells the renderer to get all the native assets from Prebid.js.
+
+Example creative HTML:
+```
+<div class="sponsored-post">
+  <div class="thumbnail" style="background-image: url(##hb_native_image##);"></div>
+  <div class="content">
+    <h1><a href="%%CLICK_URL_UNESC%%##hb_native_linkurl##" target="_blank" class="pb-click">##hb_native_title##</a></h1>
+    <p>##hb_native_body##</p>
+    <div class="attribution">##hb_native_brand##</div>
+  </div>
+</div>
+<script src="https://cdn.jsdelivr.net/npm/prebid-universal-creative@latest/dist/native-render.js"></script>
+<script>
+    var pbNativeTagData = {};
+    pbNativeTagData.pubUrl = "%%PATTERN:url%%";
+    pbNativeTagData.adId = "%%PATTERN:hb_adid%%";
+    // if you're using 'Send All Bids' mode, you should use %%PATTERN:hb_adid_BIDDER%%
+    pbNativeTagData.requestAllAssets = true;
+    window.pbNativeTag.renderNativeAd(pbNativeTagData);
+</script>
+```
```

Comment:
Hi @bretg 

When I was looking at this page on the local build, it seems the formatting around this code block isn't working as expected which affected the following code/formatting down to the next major heading.  I attached a screenshot for reference:
![Screen Shot 2021-01-13 at 8 42 33 AM](https://user-images.githubusercontent.com/31102355/104460116-ac00bd00-557b-11eb-9ff8-c3eaf9aaac3e.png)

Could you take a look?

Reply:
@jsnellbaker - I don't see this mis-formatting on my local Jekyll. And the markdown looks ok. Am going to go ahead and merge, will check the site and fix if needed.